api/src/services/apps_service.rs

```diff
@@ -58,10 +61,70 @@ impl AppsService {
         })
     }
 
+    fn resolve_web_host_meta(
+        app_name: &String,
+        service_name: &String,
+        endpoint_url: &Url,
+        request_info: &RequestInfo,
+    ) -> Option<WebHostMeta> {
+        let url = endpoint_url.join(".well-known/host-meta.json").unwrap();
+
+        let get_request = reqwest::Client::builder()
+            .build()
+            .unwrap()
+            .get(url)
+            .header(
+                "Forwarded",
+                format!(
+                    "host={};proto={}",
+                    request_info.host(),
+                    request_info.scheme()
+                ),
+            )
+            .header(
+                "X-Forwarded-Prefix",
+                format!("/{}/{}", app_name, service_name),
```

Comment:
One _might_ add recursive Proxy support here (prepending any incoming `X-Forwarded-Prefix`).
Maybe this is premature, but somehow I get the feeling it'll be useful sooner or later.

Reply:
Do you think of scenarios in which PREvant is running behind another proxy?

Reply:
If so, do you want to add an issue so that we can keep track of it?

Reply:
Good call, I opened #4 for this